test/integration/full/configure-options/configure-options.js

```diff
@@ -203,7 +203,7 @@ describe('Configure Options', function () {
       };
 
       var iframe = document.createElement('iframe');
-      iframe.src = '/test/mock/frames/context.html';
+      iframe.src = '/test/mock/frames/noHtml-config.html';
```

Comment:
The original test was `configure`ing in the top frame but not the child frame. This isn't supported and doesn't provide the protection `noHtml` is intended to (the thing that the test after this one is verifying). However, previously, the original test worked despite testing an unsupported configuration because there was an extra round trip through `DqElement` in the top frame involved that no longer exists.

I think it's preferable to break the old pattern and demand that configure must be called consistently in each frame, so I changed the test rather than forcing a top-frame-only `noHtml` configure to suppress `html` properties from child frame results, but if that feels too concerning for backcompat reasons you could address it by adding an `audit.noHtml` test into the `process-aggregate.js` code that's filling in `html` from `node.source`.